docs/report/report.tex

```diff
@@ -43,6 +43,36 @@ \subsubsection{Stretch Goals}
   \item Cloud-Anbindung der Box, Auslieferung des UI aus der Cloud
 \end{enumerate}
 
+\section{Projektplanung}
+
+\subsection{Projekt Management}
+\colorbox{red}{ticket pool in github, alles zentral, 3x daily standup, gemeinsamer google calendar, interne team evaluation und feedback}
+
+\subsection{Projektrisiken}
+\colorbox{red}{Welche Riskiken hatten wir + Risikomatrix? lohnt sich dieser Abschnitt?}
+
+\subsection{Phasenplanung}
+\subsubsection{Projektstartphase}
+\colorbox{red}{Github aufsetzen, Tickets befüllen, Anforderungen verstehen, Team einarbeiten in Technologien}
+
+\subsubsection{Realisierungsphase}
+\colorbox{red}{Coden etc}
+
+\subsubsection{Testphase}
+\colorbox{red}{Tests halt, bau mal alles zusammen undso. Wurde am Ende aber auch mehr Hand in Hand mit der vorherigen gemacht}
```

Comment:
```suggestion
\subsubsection{Realisierungsphase}
\colorbox{red}{Iteratives Arbeiten in Meilensteinen/Sprint mit Länge ca. 2 Wochen. Coden etc. Jedes Arbeitspaket wird einzeln getestet und von mindestens zwei anderen Personen reviewt (Technische Funktionalität wird automatisch getestet)}
```
Ich stimme zu, dass das Testen und Coden Hand in Hand ging und würde dementsprechend auch die Berichtsabschnitte zusammenlegen. Ich würde auch sagen, dass wir rausstellen dass wir im Hauptteil iterativ arbeiten (Sprints)